exercises/tic-tac-toe.livemd

```diff
@@ -155,8 +155,20 @@ defmodule TicTacToe do
     "C"
   """
   def at(board, coordinate) do
+    y = Kernel.elem(coordinate, 1)
+    x = Kernel.elem(coordinate, 0)
```

Comment:
Consider pattern matching.

```elixir
{x, y} = coordinate
```

Reply:
Fortunately, now I know what this means. Thanks!

Reply:
I think I have it...